deploy/olm/csv-rhel/storageos.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,355 @@
+    500GB after registration.  For additional capacity and support plans contact
+    sales@storageos.com.
+
+    ## Supported Features
+
+    * **Rapid volume failover** - If a container gets re-scheduled to another
+    node, any StorageOS volumes can be re-attached immediately, irrespective of
+    where the data is located within the cluster.
+
+    * **Data protection** - Data is replicated synchronously, up to 6 copies.
+
+    * **Block checksums** - Each block is protected by a checksum which
+    automatically detects any corruption of data in the underlying storage
+    media.
+
+    * **Thin provisioning** - Only consume the space you need in a storage pool.
+
+    * **Data reduction** - Transparent inline data compression to reduce the
+    amount of storage used in a backing store as well as reducing the network
+    bandwidth requirements for replication.
+
+    * **In-memory caching** - Speed up access to data even when accessed
+    remotely.
+
+    * **Quality of service** - Prioritize data traffic and address the “noisy
+    neighbors” problem.
+
+    * **Flexible configuration** - Use labels to automate data placement and
+    enforce data policy such as replication. Ideal for compliance and infosec
+    teams to enforce policies and rules while still enabling self-service
+    storage by developers and DevOps teams.
+
+    * **Access control** - Support multiple namespace – individual users are
+    permissioned to specific storage namespaces.
+
+    * **Observability & instrumentation** - Log streams for observability and
+    Prometheus support for instrumentation.
+
+    * **Deployment flexibility** - Scale up or scale out storage based on
+    application requirements.  Works with any infrastructure – on-premises, VM,
+    bare metal or cloud.
+
+    * **Small footprint** - Lightweight container requires minimum 1 core with
+    2GB of RAM.  Runs in user-space on any 64-bit Linux with no custom kernel
+    modules.
+
+    ## Required Parameters
+
+    * `secretRefName` - the name of a secret that contains two keys for the `apiUsername` and `apiPassword` to be used as api credentials ([documentation](https://docs.storageos.com/docs/reference/cluster-operator/examples))
+    * `secretRefNamespace` - the namespace where the api credentials secret is stored
+
+    ## Required SCC
+
+    Make sure to install the Security Context Constraint (SCC) in order for
+    the StorageOS DaemonSet and StatefulSet pods to start properly:
+
+    ``` $ oc adm policy add-scc-to-user privileged system:serviceaccount:<namespace>:storageos-daemonset-sa ```
+
+    ``` $ oc adm policy add-scc-to-user privileged system:serviceaccount:<namespace>:storageos-statefulset-sa ```
+
+    By default, the StorageOS cluster is deployed in `storageos` namespace. The
+    service accounts in the above commands should be
+    `:storageos:storageos-daemonset-sa` and `:storageos:storageos-statefulset-sa`.
```

Comment:
SCC requirements for openshift.

Reply:
Maybe we should add this to the upstream k8s CSV as well and mention that this is a requirement for openshift only.